README.md

```diff
@@ -24,40 +24,8 @@ This action will start, stop or restart a Windows service on an on-prem Windows
 
 ## Prerequisites
 
-The windows service action uses Web Services for Management, [WSMan], and Windows Remote Management, [WinRM], to create remote administrative sessions. Because of this, Windows OS GitHubs Actions Runners, `runs-on: [windows-2019]`, must be used. If the file deployment target is on a local network that is not publicly available, then specialized self hosted runners, `runs-on: [self-hosted, windows-2019]`,  will need to be used to broker deployment time access.
-
-Inbound secure WinRm network traffic (TCP port 5986) must be allowed from the GitHub Actions Runners virtual network so that remote sessions can be received.
-
-Prep the remote Windows server to accept WinRM management calls.  In general the Windows server needs to have a [WSMan] listener that looks for incoming [WinRM] calls. Firewall exceptions need to be added for the secure WinRM TCP ports, and non-secure firewall rules should be disabled. Here is an example script that would be run on the Windows server:
-
-  ```powershell
-  $Cert = New-SelfSignedCertificate -CertstoreLocation Cert:\LocalMachine\My -DnsName <<ip-address|fqdn-host-name>>
-
-  Export-Certificate -Cert $Cert -FilePath C:\temp\<<cert-name>>
-
-  Enable-PSRemoting -SkipNetworkProfileCheck -Force
-
-  # Check for HTTP listeners
-  dir wsman:\localhost\listener
-
-  # If HTTP Listeners exist, remove them
-  Get-ChildItem WSMan:\Localhost\listener | Where -Property Keys -eq "Transport=HTTP" | Remove-Item -Recurse
-
-  # If HTTPs Listeners don't exist, add one
-  New-Item -Path WSMan:\LocalHost\Listener -Transport HTTPS -Address * -CertificateThumbPrint $Cert.Thumbprint –Force
-
-  # This allows old WinRm hosts to use port 443
-  Set-Item WSMan:\localhost\Service\EnableCompatibilityHttpsListener -Value true
-
-  # Make sure an HTTPs inbound rule is allowed
-  New-NetFirewallRule -DisplayName "Windows Remote Management (HTTPS-In)" -Name "Windows Remote Management (HTTPS-In)" -Profile Any -LocalPort 5986 -Protocol TCP
-
-  # For security reasons, you might want to disable the firewall rule for HTTP that *Enable-PSRemoting* added:
-  Disable-NetFirewallRule -DisplayName "Windows Remote Management (HTTP-In)"
-  ```
-
-- `ip-address` or `fqdn-host-name` can be used for the `DnsName` property in the certificate creation. It should be the name that the actions runner will use to call to the Windows server.
-- `cert-name` can be any name.  This file will used to secure the traffic between the actions runner and the Windows server
+1. The target windows machine that will be running the service will need to have a WinRM SSL listener setup.  This will have to be setup through a service ticket because a specifically formatted SSL certificate will need be set up in the correct certificate container.
+2. A deployment service principal will need to be created and put into the local admins group of the target server.
```

Comment:
How does one do this? Is it also a ticket, or something in terraform if the machine is in azure? This might be a dumb question and not need to be answered in the README, but I want to ask for my own edification at least.

Reply:
That's a good question.  It's through a service ticket, I'll put that in.